OSDOCS-4928: Docs for cert-manager operator GA (#56754)
openshift/openshift-docs: modules/cert-manager-install-console.adoc

Reviewer: Most of my comments on this file are about formatting. Let me know if you have any questions.

. Select the *{cert-manager-operator}* and click *Install*.

> . On the *Install Operator* page:

Reviewer: ```suggestion
. On the Install Operator page:
```

I don't think we need to highlight page names. In this particular case, the page name uses headline-style capitalization, so that should suffice.

Reviewer 2: Page names are always bolded

Reviewer: Thanks for letting me know! That isn't the case for IBM Style, and I checked the SSG, but didn't see anything there about it. I'll remember that going forward.

> .. The *Update channel* defaults to *stable-v1*, which installs the latest stable release of the {cert-manager-operator}.

Reviewer: ```suggestion
.. Ensure that the update channel is set to the default setting of *stable-v1*, which installs the latest stable release of the {cert-manager-operator}.
```

Start each step in a procedure with an imperative verb.

> .. The *Installation Mode* is set to *A specific namespace on the cluster*.

Reviewer: ```suggestion
.. Ensure that the Installation Mode is set to *A specific namespace on the cluster*.
```

Author: Removed this sentence all together since there is no action needed on the users part here anymore

> .. Choose the *Installed Namespace* for the Operator. The default Operator recommended namespace is `cert-manager-operator`.

Reviewer: ```suggestion
.. Choose the Installed Namespace for the Operator. The default Operator namespace is `cert-manager-operator`.
```

. Select the *{cert-manager-operator}* and click *Install*.

. On the *Install Operator* page:
.. The *Update channel* is set to *stable-v1*, which installs the latest stable release of the {cert-manager-operator}.
+

> // TODO: Right now I see "alpha", "stable-v1", "stable-v1.10", and tech-preview. Need to see if all of these will be available at GA or not, and if any shown should be avoided / descriptions provided

Author: @thejasn @swghosh Will all of these channels be available at GA? Is one of them preferred over the others or are there any that should be avoided?

Reviewer 3: Recommended would be `stable-v1`

Author: Thanks!

.. The *Installation Mode* is set to *A specific namespace on the cluster*. This mode installs the Operator in the Operator-recommended `cert-manager-operator` namespace to watch and be made available to all namespaces in the cluster.
+
// TODO: See if above description needs adjusted. Maybe delete the part about installing in the ns, because that is covered later
.. Choose the *Installed Namespace* for the Operator. The default Operator recommended namespace is `cert-manager-operator`.
+

> // TODO: Confirm that the namespace was purposefully changed. Also check on the OLM files that are using "openshift-cert-manager-operator" to see if that is affected (Alex Greene)

Author: Hi @awgreene! The namespace that the cert-manager operator is being installed to is changing with our GA release. 

I saw several references to the old namespace of "openshift-cert-manager-operator" on this platform operators page: https://docs.openshift.com/container-platform/4.12/operators/admin/olm-managing-po.html

Two references are as a namespace, and the other is some sort of list of supported operators (but i'm not sure where this value comes from). 

Do you know if any of those references should also be changed with this namespace name change?

Reviewer 4: @bergerhoffer "openshift-cert-manager-operator" is the package name of the Operator. See https://docs.openshift.com/container-platform/4.12/operators/understanding/olm-packaging-format.html#olm-file-based-catalogs_olm-packaging-format about the "name" within "olm.package". You can see:
$ oc get packagemanifest -n openshift-marketplace -o yaml
```
    defaultChannel: stable-v1
    packageName: openshift-cert-manager-operator
```
CC @thejasn @swghosh .

Author: @xingxingxia Thanks! So that means that the OLM docs are okay? We don't even need to update those namespace references in the OLM docs?

Reviewer 4: 1 For occurrences that reference "openshift-cert-manager-operator" as package name, need not update.
2 For occurrences that reference "openshift-cert-manager-operator" as namespace name, we should update for which seems you are already done.
